### ISSUE TITLE
The torch. nn. MarginRankingLoss function documentation errors

### ISSUE THREAD

**Issue by laizewei** [pytorch/pytorch#14226]
## 📚 Documentation
The torch. nn. MarginRankingLoss function documentation errors. I run in pycharm:
 loss = x2 * (x2 - x1) + margin
--------------------------------------------------------
import torch as torch
import torch.nn as nn
a=[2,1,1,1]
b=[2,-3,1,1]
c=[-1,1,-1,-1]
input = torch.Tensor(a)
input=input.view(4)
inputa = torch.Tensor(b)
inputa=inputa.view(4)
inputc = torch.Tensor(c)
inputc=inputa.view(4)
loss=nn.MarginRankingLoss(margin=1,reduce=True,size_average=False)
lossa=loss(input,inputa,inputc)
print ("firstvar1",lossa)
-----------------
firstvar1 tensor(16.)
